src/main/java/net/sf/mzmine/parameters/parametertypes/ranges/DoubleRangeComponent.java

```diff
@@ -73,8 +74,12 @@ public void setNumberFormat(NumberFormat format) {
     }
 
     public void setValue(Range<Double> value) {
-        minTxtField.setText(format.format(value.lowerEndpoint()));
-        maxTxtField.setText(format.format(value.upperEndpoint()));
+        NumberFormat floorFormat = (NumberFormat) format.clone();
+        floorFormat.setRoundingMode(RoundingMode.FLOOR);
+        NumberFormat ceilFormat = (NumberFormat) format.clone();
+        ceilFormat.setRoundingMode(RoundingMode.CEILING);
+        minTxtField.setText(floorFormat.format(value.lowerEndpoint()));
+        maxTxtField.setText(ceilFormat.format(value.upperEndpoint()));
     }
 
```

Comment:
Won't this affect the value of all range parameters set in any dialog window - e.g. also if a user specifies their own range?